configs/server/forseti_conf_server.yaml.in

```diff
@@ -1,41 +1,41 @@
 global:
 
-    # database
-    db_host: 127.0.0.1
-    db_user: root
-    db_name: forseti_security
-
-    # gsuite
-    groups_service_account_key_file: /home/ubuntu/gsuite_key.json
-    domain_super_admin_email: {DOMAIN_SUPER_ADMIN_EMAIL}
-
-    # notification
     # Remove or comment out the email properties if you do not want Forseti to
     # send email notifications.
     email_recipient: {EMAIL_RECIPIENT}
     email_sender: {EMAIL_SENDER}
     sendgrid_api_key: {SENDGRID_API_KEY}
 
-    # quota
+##############################################################################
+
+inventory:
+
+    # Root resource to start crawling from, formatted as
+    # <resource_type>/<resource_id>, (e.g. "organizations/12345677890")
+    root_resource_id: {ROOT_RESOURCE_ID}
+
+    # gsuite access
+    groups_service_account_key_file: /home/ubuntu/gsuite_key.json
+    domain_super_admin_email: {DOMAIN_SUPER_ADMIN_EMAIL}
+
+    api_quota:
     # The pre-populated values are the defaults from GCP.
```

Comment:
really small nit, not a blocker: seems like this line should be shift too

Reply:
Done.